Assets/MRTK/SDK/Features/UX/Scripts/PulseShader/SurfacePulse.cs

```diff
@@ -8,20 +8,21 @@
 using UnityEngine;
 using UnityEngine.Serialization;
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
+namespace Microsoft.MixedReality.Toolkit.UI.SurfacePulse
 {
     /// <summary>
-    /// Script for generating pulse shader effect on the surface.
+    /// Script for generating pulse shader effect on the surface of a mesh.  
     /// </summary>
     [AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
     public class SurfacePulse : MonoBehaviour, IMixedRealityPointerHandler
```

Comment:
Not new in this PR, but this MonoBehaviour has an explicit constructor, which isn't really a recommended pattern. Could the contents be moved to `Awake`/`Start`?

Reply:
We should also make sure we un-register from the event handler that's in the constructor

Reply:
Removed the constructor